psutil/tests/test_osx.py

```diff
@@ -144,6 +146,7 @@ def test_cpu_count_cores(self):
         num = sysctl("sysctl hw.physicalcpu")
         self.assertEqual(num, psutil.cpu_count(logical=False))
 
+    @expectedFailureIf(MACOS_ARM64)  # macOS arm64 not supported: issue #1892
```

Comment:
See my comment above: there's no need of an additional decorator. Please define this as:

```python
# TODO: remove this once 1892 is fixed
@unittes.skipIf(MACOS and platform.machine() == 'arm64', "skipped due to #1892")
def test_cpu_freq(self):
    ...
```